### ISSUE TITLE
Ablation on lr vs. gradient norm clipping

### ISSUE THREAD

**Issue by Jubeku** [ecmwf/WeatherGenerator#544]
### Is your feature request related to a problem? Please describe.

Question was whether we can increase the learning rate with a more conservative gradient norm clipping value. 

### Describe the solution you'd like

Run ablation experiments with following parameters:
- `lr_max: 5e-5, grad_clip: 5.0`
- `lr_max: 5e-5, grad_clip: 1.0`
- `lr_max: 2e-4, grad_clip: 5.0`
- `lr_max: 2e-4, grad_clip: 1.0`

### Organisation

ECMWF

**Comment by Jubeku** [comment]
<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/677db1e5-2049-45a8-9562-19f1b4a7576b" />

<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/168bb2e5-d686-4558-8366-28b14dee1135" />

<img width="3000" height="2100" alt="Image" src="https://github.com/user-attachments/assets/44f89510-136c-40ea-86e3-1b0043c0078a" />

Note: Run `g1xvezhj` not fully terminated at the time of plotting.

**Comment by Jubeku** [comment]
After discussing within the ERA5 performance meeting, we concluded that a learning rate of `2e-4` might be slightly too big.
